README.md

```diff
@@ -1,99 +1,66 @@
-# Quay Installer
-
-This application will allow user to install Quay and its required components using a simple CLI tool.
+# OpenShift Mirror Registry
 
+This application will allow user to easily install Quay and its required components using a simple CLI tool. The purpose is to provide a registry to hold a mirror of OpenShift images.
 ## Pre-Requisites
 
-- RHEL 8 or Fedora machine with Podman installed
-- `sudo` access on desired host (rootless install tbd)
-- make (only if compiling using Makefile)
-- sshd must be enabled
-
-### Compile
-
-To compile the openshift-mirror-registry.tar.gz for distribution, run the following command:
-
-```console
-$ git clone https://github.com/quay/openshift-mirror-registry.git
-$ cd openshift-mirror-registry
-$ make build-online-zip # OR make build-offline-zip
-```
-
-This will generate a `openshift-mirror-registry.tar.gz` which contains this README.md, the `openshift-mirror-registry` binary, and the `image-archive.tar` (if using offline installer) which contains the images required to set up Quay.
-
-Once generated, you may untar this file on your desired host machine for installation. You may use the following command:
-
-```console
-mkdir openshift-mirror-registry
-tar -xzvf openshift-mirror-registry.tar.gz -C openshift-mirror-registry
-```
-
-NOTE - With the offline version, this may take some time.
-
+- RHEL 8 or Fedora machine with `podman` installed
+- fully qualified domain name for the Quay service (must resolve via DNS, or at least [/etc/hosts](#local-dns-resolution))
+- passwordless `sudo` access on the target host (rootless install tbd)
+- key-based SSH connectivity on the target host (will be set up automatically for local installs, in case of remote hosts see [here](#generate-ssh-keys))
+- `make` (only if [compiling](#compile-your-own-installer) your own installer)
 ## Installation
 
-Add the following line to host machine `/etc/hosts` file:
-
-```
-<targetHostname ip>   quay
-```
-
-### SSH Keys
-
-In order to run the installation playbooks, you must also set up SSH keys. Local installations will automatically generate the SSH keys for you.
-
-To generate your own SSH keys to install on a remote host, run the following commands.
-
-```console
-$ ssh-keygen
-$ ssh-add
-$ ssh-copy-id <targetHostname>
-```
-
-You can provide your ssh private key to the installer CLI with the --ssh-key flag.
+Download one of the installer package from our [releases](https://github.com/quay/openshift-mirror-registry/releases) page:
 
+- online version (additional container images to run Quay and Postgres will be downloaded by the installer)
+- offline version (contains all required images to run Quay)
 ### Running the installer
 
-To install Quay on localhost, run the following command:
+To install Quay on your local host, run the following command:
 
 ```console
-$ ./openshift-mirror-registry install -v
+$ ./openshift-mirror-registry install --quayHostname $(hostname -f) -v
```

Comment:
We can resolve the --quayHostname value automatically during the installation with the $HOST default env variable, so this isn't necessary to include for local host installations.